API_request_test.py

```diff
@@ -87,4 +87,49 @@
 print(hours_2)
 
 print(temp_c)
-print(temp_c_2)
\ No newline at end of file
+print(temp_c_2)
+
+
+#####################
+### meteostat.net ###
+#####################
+
+# You need to register on the web page first to get API key
+# https://rapidapi.com/meteostat/api/meteostat/pricing
+# The url is set to get forecast data. For historical data it should be changed
+
+url_3 = 'https://meteostat.p.rapidapi.com/point/hourly'
+API_KEY_3 = <paste here your personal API key>
+
+params_3 = {
+	"lat": lat,
+	"lon": lon,
+	"start":"2024-02-28", #here we have to use some timestump to do it automaticaly
+    "end":"2024-02-28"
+}
+
+headers_3 = {
+	"X-RapidAPI-Key": <paste here your personal API key>,
+	"X-RapidAPI-Host": "meteostat.p.rapidapi.com"
+}
+
+res_3 = requests.get(url_3, headers= headers_3, params= params_3)
```

Comment:
Please be sure to use the recommended Python code style guide and omit the whitespace after function parameter assignment (see [PEP rules](https://peps.python.org/pep-0008/#other-recommendations) for a reference). To make it easier for the future, you can install [a linter extension](https://code.visualstudio.com/docs/python/linting) in your IDE.

Reply:
Done.